### DISCUSSION TITLE
Creating a Figma collection for all components

### DISCUSSION THREAD

**Discussion by fcastillo18** [cosscom/coss#62] [Suggestions]
I think having a Figma repository/collection with all these amazing components will be awesome.

What do you guys think?

**Comment by davidepacilio** [comment]
Hey @fcastillo18, currently, we have no plans to support Origin UI in Figma. However, you can find a non-official version here: [https://figma.com/community/file/1432728348412608289…](https://t.co/OU9xyU08Fg). Please note that it may not be consistently maintained.

**Reply by fcastillo18** [reply]
Hi @davidepacilio! Thanks for the fast response. Yeah, I understand that maintaining a Figma will be hard with the changes and new components you guys are adding. 

Thanks for sharing that non-official version, will be a good start.

Thank you and I appreciate all the effort in this project, looks very promising!!! Keep it rocking :)
